src/app/clusters/resource-monitoring-server/resource-monitoring-cluster-objects.h

```diff
@@ -29,7 +29,10 @@ namespace app {
 namespace Clusters {
 namespace ResourceMonitoring {
 
-static constexpr std::array<ClusterId, 2> AliasedClusters = { 0x0071, 0x0072 };
+#define HEPA_FILTER_CLUSTER_ID 0x0071
+#define ACTIVATED_CARBON_FILTER_CLUSTER_ID 0x0072
+
+static constexpr std::array<ClusterId, 2> AliasedClusters = { HEPA_FILTER_CLUSTER_ID, ACTIVATED_CARBON_FILTER_CLUSTER_ID };
```

Comment:
like the idea, but would not redefine it here, just use the ones defined in clusters.h ?
```suggestion
static constexpr std::array<ClusterId, 2> AliasedClusters = { HepaFilterMonitoring::Id, ActivatedCarbonFilterMonitoring::Id };
```